pallas-txbuilder/src/asset.rs

```diff
@@ -0,0 +1,54 @@
+use std::marker::PhantomData;
+
+use indexmap::IndexMap;
+use pallas_codec::utils::{Bytes, KeyValuePairs};
+use pallas_primitives::babbage::PolicyId;
+
+#[derive(Debug, Clone, PartialEq, Eq, thiserror::Error)]
+pub enum AssetError {
+    #[error("Invalid asset name")]
+    InvalidAssetName(String),
+}
+
+#[derive(Debug, Clone, Default)]
+pub struct MultiAsset<T> {
+    assets: IndexMap<PolicyId, Vec<(Bytes, T)>>,
+    _marker: PhantomData<T>,
```

Comment:
You can probably remove the marker, right? Since `assets` will have a concrete `T` at compilation.

Reply:
True! Will do that.

Reply:
Fixed!